### PR TITLE
Refactor Readme

### PR DESCRIPTION
**Pull request by masskoder**
This PR includes the refactoring of the table status pill, which was pushed to `rotationalio/beacon-core@2.5.16` a few weeks ago. Additionally, I've revamped the README for better clarity.

@daniellemaxwell  If everything looks good to you, feel free to merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,388 @@
...
 ```
 
-## System Requirements
+4.Start the development mode:
 
-- git
-- NodeJS `>=16.0.0`
-- yarn v3
+```bash
+yarn dev
```
Comment (daniellemaxwell) [inline README.md:384]: Is this the correct command? It works to run the docs in the `/apps/docs`directory, but I don't see `dev` listed as a script in any other `package.json` files. Also, running `yarn dev` returns the following error in the root:
`Couldn't find a script named "dev".`

File: README.md [hunk 2]
```diff
@@ -1,45 +1,388 @@
...
+```
 
 ## Run Storybook
```
Comment (daniellemaxwell) [inline README.md:387]: After entering the command `yarn storybook`, the following error appears multiple times and an error stating that the build is broken appears:
```
Field 'browser' doesn't contain a valid alias configuration
packages/core/lib/tailwind.css.cjs doesn't exist
      as directory
```

File: README.md (hunk 1 above)
Comment (masskoder) [reply README.md:384]: Nice catch! Sometimes I get a bit confused when working on two documents, but I'll make sure to update that. Thanks!

File: README.md (hunk 2 above)
Comment (masskoder) [reply README.md:384]: This error occurred when we transitioned from Yarn Berry 3.2.1 to 3.4.1. Please try to delete the 'lib' folder in the core package and rebuild it.

Comment (daniellemaxwell) [reply README.md:384]: Unfortunately, that didn't work. I'll continue to investigate when I have time.

**Pull request merged**